### DISCUSSION TITLE
Pasteleria

### DISCUSSION THREAD

**Discussion by DiegoCorchuelo** [Launch-X-Latam/MisionFrontEnd#260] [Show and tell]
He terminado la practica de Html, Me gustaría saber sus opiniones y consejos.
https://diegocorchuelo.github.io/Pasteleria.github.io/Pastelero.html
https://diegocorchuelo.github.io/Pasteleria.github.io/Cliente.html

**Comment by brayandoria98** [comment]
Está muy bien, me gusta como quedó. Es lo que se pide en las prácticas y me sacaste de dudas porque yo también pensé que era dos páginas. Una para el cliente y otra para el pastelero, pero no estaba seguro. Gracias.

**Reply by DiegoCorchuelo** [reply]
Gracias a ti por tu comentario.
